drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c

```diff
@@ -2355,6 +2369,9 @@ static int bcm2835_codec_open(struct file *file)
 					 BIT(V4L2_MPEG_VIDEO_H264_PROFILE_MAIN) |
 					 BIT(V4L2_MPEG_VIDEO_H264_PROFILE_HIGH)),
 					V4L2_MPEG_VIDEO_H264_PROFILE_HIGH);
+		v4l2_ctrl_new_std(hdl, &bcm2835_codec_ctrl_ops,
```

Comment:
In adding an extra control, the size hint in the call ```v4l2_ctrl_handler_init(hdl, 6);``` above ought to be incremented by 1, to 7.

Reply:
Fixed!